packages/core/src/components/axes/tool-bar.ts

```diff
@@ -0,0 +1,325 @@
+// Internal Imports
+import { Component } from "../component";
+import { Events } from "../../interfaces";
+import { Tools } from "../../tools";
+import { DOMUtils } from "../../services";
+import { ChartModel } from "../../model";
+import * as Configuration from "../../configuration";
+
+// D3 Imports
+import { event, select } from "d3-selection";
+
+// import the settings for the css prefix
+import settings from "carbon-components/es/globals/js/settings";
+
+export class ToolBar extends Component {
+	type = "tool-bar";
```

Comment:
```suggestion
	type = "toolbar";
```

Reply:
Updated in https://github.com/carbon-design-system/carbon-charts/pull/717/commits/bc342d7f9a579d5142cfca38947870f8cd2de4e2.